applications/StructuralMechanicsApplication/tests/cpp_tests/test_mpc_plate_tension.cpp

```diff
@@ -0,0 +1,164 @@
+#include <iostream>
```

Comment:
The header is missing

Reply:
i did not get what you mean?

Reply:
~~~cpp
// KRATOS  ___|  |                   |                   |
//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
//             | |   |    |   | (    |   |   | |   (   | |
//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
//
//  License:             BSD License
//                                       license: StructuralMechanicsApplication/license.txt
//
//  Main authors:    YOUR NAME HERE
//
~~~

Reply:
done in 31b5debeb4808262370ed9575b2059030b8b639f